src/Tizen.Multimedia.Vision/MediaVision/QrConfiguration.cs

```diff
@@ -78,5 +82,67 @@ public QrConfiguration(QrMode qrMode, ErrorCorrectionLevel ecc, int version)
         /// </summary>
         /// <since_tizen> 4 </since_tizen>
         public int Version { get; }
+
+        /// <summary>
+        /// Gets or sets the embed image absolute path of the Design QR code.
```

Comment:
is "Design QR" some special term?  "embed" is a verb so I'm not sure this is correct. Please describe to me how does this work and I will present some suggestions for rewording. While we are at it, you could explain that Data and Finder shapes are.

Reply:
> is "Design QR" some special term? 

Yes it is. It just 
> "embed" is a verb so I'm not sure this is correct. Please describe to me how does this work and I will present some suggestions for rewording. While we are at it, you could explain that Data and Finder shapes are.

EmbedImagePath is kinds of the logo image that goes in the middle of the QR code.
you could see more deatil in here
https://github.com/Samsung/tizen-docs/blob/tizen_7.5_prepare/docs/application/native/guides/multimedia/image-barcode.md#design-qr